src/interfaces/assistants_web/src/stores/slices/citationsSlice.ts

```diff
@@ -1,7 +1,7 @@
 import { StateCreator } from 'zustand';
 
 import { Document } from '@/cohere-client';
-import { mapExtensionToMimeType } from '@/utils';
+import { decodeBase64, mapExtensionToMimeType } from '@/utils';
```

Comment:
we can use native browser API for these operations:

https://developer.mozilla.org/en-US/docs/Web/API/Window/atob

Reply:
I am getting contradictory information over

<img width="1485" alt="image" src="https://github.com/user-attachments/assets/f1d7fd29-3b29-4a04-a385-6d2d760d92ed">